cmd/ddev/cmd/commands.go

```diff
@@ -542,3 +559,14 @@ func findDirectivesInScriptCommand(script string) map[string]string {
 
 	return directives
 }
+
+func runMutagenSync(app *ddevapp.DdevApp, mutagenSync bool) {
+	if mutagenSync {
+		if status, _ := app.SiteStatus(); status == ddevapp.SiteRunning {
+			err := app.MutagenSyncFlush()
+			if err != nil {
+				util.Failed("failed to app.MutagenSyncFlush: %v", err)
```

Comment:
I suspect a `util.Warning()` would be gentler

```suggestion
				util.Warning("failed to app.MutagenSyncFlush: %v", err)
```